pkg/vulnsrc/osv/osv.go

```diff
@@ -350,6 +350,8 @@ func convertEcosystem(eco Ecosystem) types.Ecosystem {
 		return vulnerability.Swift
 	case "bitnami":
 		return vulnerability.Bitnami
+	case "k8s":
```

Comment:
It must be the same value used in OSV advisories.

```suggestion
	case "kubernetes":
```